# Request (input del service)
---
bool is_available  # Response (output del service)